frontend/src/app/styles/app.css

```diff
@@ -0,0 +1,5 @@
+.course-input-container {
+  max-width: 50rem;
+  margin: 40px auto; }
```

Comment:
define some css variables or margins and paddings, they should usually be multiple of 2. Also keep them in rem, makes the layout more responsive and less prone to ui bugs